tools/travis/run_tests.sh

```diff
@@ -27,6 +27,10 @@ fi
 
 test="$1"
 
+if [ "$#" -gt 1 ]; then
```

Comment:
On a second look, I think we should have

```
if [ "$#" -gt 2]; then
  echo "Usage: "$0" <test> [PATTERN]"
  exit 1
fi
```

And update the message in case of no arguments as well.